project-api/project-api-test/src/test/resources/old_dist_version_project/Ballerina.toml

```diff
@@ -0,0 +1,5 @@
+[package]
+org = "sameera"
+name = "myproject"
+version = "0.1.0"
+distribution = "slbeta1"
```

Comment:
We are adding the distribution version from `beta4`. Shall we update this test case to not have the distribution version? This is related to https://github.com/ballerina-platform/ballerina-lang/pull/33723/files#r746322422